src/OpenTelemetry.Extensions/Internal/OpenTelemetryExtensionsEventSource.cs

```diff
@@ -44,4 +44,10 @@ public void LogRecordFilterException(string? categoryName, string exception)
     {
         this.WriteEvent(2, categoryName, exception);
     }
+
+    [Event(3, Message = "Baggage key predicate function threw exeption: '{0}'", Level = EventLevel.Error)]
```

Comment:
```suggestion
    [Event(3, Message = "Baggage key predicate function threw exeption: '{0}'", Level = EventLevel.Warning)]
```

Reply:
Error is better suited for more serious conditions like telemetry loss etc. This one is less severe, as the telemetry will continue to flow without issues, just that the baggage won't be attached.

Reply:
Also - could we make the message more clear, to indicate the impact of this? Will this particular key be not added? or entire baggage?

Reply:
Updated message in [bc08c17](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1816/commits/bc08c17ae39a79a6b87c282d6e109ad151e47142)